resources/release/missing-code-coverage.md

```diff
@@ -0,0 +1,6 @@
+Hi, </br>
+
+This component is not reporting code-coverage for branch [$BRANCH]($CODECOV_URL). </br>
```

Comment:
Is it possible to insert the component name?

Reply:
We are directly commenting on the component's issue. But sure just to be clear let me add it.

Reply:
Added the change here: https://github.com/opensearch-project/opensearch-build-libraries/pull/631/commits/8af4484f97d77a938b709bb46c08ea42a847930d
Thanks!